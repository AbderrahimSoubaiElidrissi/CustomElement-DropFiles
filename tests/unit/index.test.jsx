/* eslint-env node, jest */

import { h } from "preact";
import render from "preact-render-to-string";
import DropFiles from "../../src/components/drop-files/index";

describe("DropFiles Snapshot", () => {
  it("should render DropFiles with content", () => {
    const tree = render(<DropFiles />);
    expect(tree).toMatchSnapshot();
  });
});
