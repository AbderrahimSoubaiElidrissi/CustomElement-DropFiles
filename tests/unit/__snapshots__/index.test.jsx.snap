// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropFiles Snapshot should render DropFiles with content 1`] = `"<div class=\\"drop-files\\"><input type=\\"file\\" multiple style=\\"display:none;\\" /><div class=\\"drop-files__files\\"></div><div class=\\"drop-files__explanations\\"><strong>Drop files here or click to upload.</strong></div><input type=\\"file\\" multiple class=\\"drop-files__fake\\" /></div>"`;
